<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTagManager</class>
 <widget class="QWidget" name="EditorTagManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>91</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="TagList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>366</width>
     <height>44</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="NewTag">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Declare a new tag</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/tag--pencil.png</normaloff>:/icons/tag--pencil.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="AddTag">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add current tag from list</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/tag--arrow.png</normaloff>:/icons/tag--arrow.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="RemoveTag">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Remove current tag</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/tag--minus.png</normaloff>:/icons/tag--minus.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="AvailableTags">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>78</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
